modules/nf-core/verifybamid/verifybamid/main.nf

```diff
@@ -0,0 +1,53 @@
+process VERIFYBAMID_VERIFYBAMID {
+    tag "${meta.id}"
+    label "process_single"
+
+    conda (params.enable_conda ? "bioconda::verifybamid=1.1.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/verifybamid%3A1.1.3--h5b5514e_6':
+        'quay.io/biocontainers/verifybamid:1.1.3--h5b5514e_6' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path refvcf
+
+    output:
+    tuple val(meta), path("*.log")                                  , emit: log
+    tuple val(meta), path("*.selfSM")                , optional:true, emit: selfsm
+    tuple val(meta), path("*.depthSM")               , optional:true, emit: depthsm
+    tuple val(meta), path("*.selfRG")                , optional:true, emit: selfrg
+    tuple val(meta), path("*.depthRG")               , optional:true, emit: depthrg
+    tuple val(meta), path("*.bestSM")                , optional:true, emit: bestsm
+    tuple val(meta), path("*.bestRG")                , optional:true, emit: bestrg
+    path "versions.yml"                                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args_list = args.tokenize()
```

Comment:
This splitting shouldn't be necessary, by defualt NXF will print them with spaces anyway (I assume left over from the earlier module). You can just directly put `$args` in the command.

Reply:
No? I don't believe so.